scripts/etc/init.d/rcS

```diff
@@ -787,7 +787,7 @@ do
 done
 
 # iterate through all the *.key files and add them to apt-key
-for keyfile in *.key
+for keyfile in $(ls *.key)
```

Comment:
This is busybox POSIX sh and file globbing does not work if there are no files to match.  Doing this '$(ls *.key) in POSIX sh will produce no results instead of passing along '*.key' in keyfile.